### DISCUSSION TITLE
Please help an Error. NoneType' object is not iterable

### DISCUSSION THREAD

**Discussion by Liferocka** [lllyasviel/stable-diffusion-webui-forge#764] [General]
Hi, I created a custom Model Lora SDXL. on Civit. and I imported it into the Stability Matrix. I am trying to use the SD UI forge. running it from Stablity Matrix.. and I get an error.  NoneType' object is not iterable I have tried so many ways to fix it? Please help.

**Comment by Liferocka** [comment]
I also get this in the ability native.  
![EaseUS_2024_05_26_15_13_10](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/50764172/11dacebd-2385-47c1-8bb4-fb575a075653)

**Comment by aolko** [comment]
you are using cumfy in stability matrix here, not forge
